login_server/player/friends.py

```diff
@@ -45,5 +45,5 @@ def load(self, filename):
 
     def save(self, filename):
         with open(filename, 'wt') as outfile:
-            json.dump(self.friends_dict, outfile, indent=4, sort_keys=True)
+            json.dump({str(k): v for k, v in self.friends_dict.items()}, outfile, indent=4, sort_keys=True)
```

Comment:
Is this a left-over for when the versions were per file?

Reply:
Yes. Removed. (If you try to dump a dict with mixed string / int keys it falls over).